dnf5.spec

```diff
@@ -521,6 +529,9 @@ Requires:       libdnf5%{?_isa} = %{version}-%{release}
 Requires:       libdnf5-cli%{?_isa} = %{version}-%{release}
 Requires:       dbus
 Requires:       polkit
+%if 0%{?fedora} <= 38
+Requires:       dnf-data
```

Comment:
We probably also want this for dnf5 package back, see [here](https://github.com/rpm-software-management/dnf5/pull/469/files#diff-3764aa091c8d6fcbde25e8587620de2d599df9a8a2897c372975ecfa4d6b4980L15).

Reply:
fixed